packages/snet_cli/snet_cli/commands/mpe_account.py

```diff
@@ -41,5 +40,10 @@ def transfer_in_mpe(self):
         self.transact_contract_command("MultiPartyEscrow", "transfer", [self.args.receiver, self.args.amount])
 
     def mint_token(self):
-        address = self.args.address
-        self.transact_contract_command("SingularityNetToken", "mint", [address, int(self.args.amount)])
+        rez = self.config.session_to_dict()
```

Comment:
we nned to remove this definition here , not expose it as a command in snet-cli

Reply:
Removed the cli command